.env.example

```diff
@@ -1,4 +1,4 @@
 PINECONE_API_KEY=
 PINECONE_INDEX="image-search"
 PINECONE_CLOUD="aws"
-PINECONE_REGION="us-west-2"
\ No newline at end of file
+PINECONE_REGION="us-east-1"
```

Comment:
Here and below: using `us-west-2` results in the following error:

```
PineconeBadRequestError: {"error":{"code":"INVALID_ARGUMENT","message":"Bad request: Your free plan does not support indexes in the us-west-2 region of aws. To create indexes in this region, upgrade your plan."},"status":400}
```

Changing to `us-east-1`, available to free tier users.